app/actions/remote/team.ts

```diff
@@ -229,7 +229,7 @@ export async function fetchMyTeam(serverUrl: string, teamId: string, fetchOnly =
     }
 }
 
-export const fetchAllTeams = async (serverUrl: string, page = 0, perPage = PER_PAGE_DEFAULT): Promise<{teams?: Team[]; error?: any}> => {
+export const fetchAllTeams = async (serverUrl: string, page = 0, perPage = PER_PAGE_DEFAULT): Promise<{teams?: Team[]; error?: unknown}> => {
```

Comment:
We really need to use `unknown` more. It's so much safer than `any`, and it's definitely a better "I don't know what this is or have the time to figure it out" type